portfolio.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>WebStudio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Raleway:wght@700&family=Roboto:ital,wght@0,400;0,500;0,900;1,700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./css/style.css" />
+  </head>
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <a href="/" class="logo">
+            <span class="logo__web">Web</span>
+            <span class="logo__studio">Studio</span>
+          </a>
+          <nav class="header__nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a class="nav__link" href="./index.html">Студія</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="./portfolio.html">Портфоліо</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#">Контакти</a>
+              </li>
+            </ul>
+          </nav>
+          <div class="contacts">
+            <a href="#" class="contacts__email">info@devstudio.com</a>
+            <a href="#" class="contacts__mobile">+38 096 111 11 11</a>
+          </div>
+        </div>
+      </div>
+    </header>
+    <main>
+      <section class="categories">
+        <div class="filtres">
+          <button type="button" class="button__categogories">Усі</button>
+          <button name="button" class="button__categogories">Веб-сайти</button>
+          <button name="button" class="button__categogories">Додатки</button>
+          <button name="button" class="button__categogories">Дизайн</button>
+          <button name="button" class="button__categogories">Маркетинг</button>
+        </div>
+      </section>
+      <section>
+        <div class="container">
+          <ul class="contents">
```

Comment:
this list makes sense only inside it's parent block, rename it to `%section-name%__list`, for example `categories__list`, in such way we mean that the section classname can be `categories` and the `categories__list` is its' element